docs/tutorials/extensive_kitchen_terraform.html

```diff
@@ -919,7 +919,7 @@ <h1 class="display-3">Extensive Kitchen-Terraform</h1>
 <div class="highlight"><pre class="syntax-highlight ruby"><code><span class="c1"># frozen_string_literal: true</span>
 
 <span class="n">source</span> <span class="s2">"https://rubygems.org/"</span> <span class="k">do</span>
-  <span class="n">gem</span> <span class="s2">"kitchen-terraform"</span><span class="p">,</span> <span class="s2">"~&gt; 5.7"</span>
+  <span class="n">gem</span> <span class="s2">"kitchen-terraform"</span><span class="p">,</span> <span class="s2">"~&gt; 6.1"</span>
```

Comment:
There doesn't appear to be a source change associated with this docs change 🕵🏼  Can you please make any documentation changes only in the `source` directory and then build `docs` by following the [Building Site](https://github.com/newcontext-oss/kitchen-terraform/blob/master/CONTRIBUTING.md#building-site) instructions?